src/test/java/com/joestelmach/natty/DateTest.java

```diff
@@ -24,323 +24,323 @@ public static void oneTime() {
   
   @Test
   public void testFormal() {
-    validateDate("1978-01-28", 1, 28, 1978);
-    validateDate("2009-10-10", 10, 10, 2009);
-    validateDate("1980-1-2", 1, 2, 1980);
-    validateDate("12/12/12", 12, 12, 2012);
-    validateDate("3/4", 3, 4, Calendar.getInstance().get(Calendar.YEAR));
-    validateDate("sun, 11/21/2010", 11, 21, 2010);
-    validateDate("in october 2006", 10, 1, 2006);
-    validateDate("feb 1979", 2, 1, 1979);
-    validateDate("jan '80", 1, 1, 1980);
-    validateDate("2006-Jun-16", 6, 16, 2006);
-    validateDate("28-Feb-2010", 2, 28, 2010);
-    validateDate("9-Apr", 4, 9, Calendar.getInstance().get(Calendar.YEAR));
-    validateDate("jan 10, '00", 1, 10, 2000);
+    validateDate(new Date(), "1978-01-28", 1, 28, 1978);
```

Comment:
maybe we should preserve the existing methods that proxy to the new ones by passing new Date() instead of forcing the parameter (just like you did with the parse() method)